app/src/main/java/org/cru/godtools/fragment/GlobalDashboardFragment.kt

```diff
@@ -0,0 +1,17 @@
+package org.cru.godtools.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.cru.godtools.databinding.FragmentGlobalDashboardBinding
+
+class GlobalDashboardFragment : BasePlatformFragment() {
+
+    private lateinit var binding: FragmentGlobalDashboardBinding
```

Comment:
`lateinit` for the binding in a fragment is bad because it will leak the view hierarchy after `onDestroyView()`.

I know we utilize it in MPDX, it's something that will need to change there as well, but is more involved because of how much we are currently using it